paddle/fluid/imperative/layer.cc

```diff
@@ -279,6 +314,15 @@ void OpBase::Run(const NameVarBaseMap& ins, const NameVarBaseMap& outs) {
   // Initialize output var type
   for (auto& var_pair : outs) {
     for (auto& var : var_pair.second) {
+      VLOG(3) << "===============^^^^^^^^^^^^^ " << var->Type() << " "
+              << framework::proto::VarType::LOD_TENSOR
+              << framework::proto::VarType::SELECTED_ROWS << std::endl;
+      /*if (var->MutableVar()->IsType<paddle::framework::LoDTensor>()){
+      var->SetType(framework::proto::VarType::LOD_TENSOR);}
+      else if (var->MutableVar()->IsType<paddle::framework::SelectedRows>()) {
+      var->SetType(framework::proto::VarType::SELECTED_ROWS);}
+      else {
+      PADDLE_THROW("Variable type is not in [LOD_TENSOR, SELECTED_ROWS]");}*/
```

Comment:
If you don't need it, please delete these comments

Reply:
done